src/components/AssetsPage/index.jsx

```diff
@@ -41,6 +50,23 @@ export default class AssetsPage extends React.Component {
     });
   }
 
+  onDeleteStatusAlertClose = () => {
+    this.props.clearAssetDeletion();
+
+    // do not attempt to focus if there are no assets delete buttons to focus on
+    // TO-DO: determine where the focus should go when the last asset is deleted
+    if (this.props.assetsList.length > 0) {
+      const focusElement = this.getNextFocusElementOnDelete();
+      focusElement.focus();
+    }
+  }
+
+  getNextFocusElementOnDelete() {
+    const deletedIndex = this.props.deletedAssetIndex;
```

Comment:
Nit: you could consider eliminating this variable (I don't know if you run into line-length problems)

```javascript
const assetToDelete = this.props.assetsList[this.props.deletedAssetIndex];
```

If you run into line-length issues, maybe

```javascript
const { deletedAssetIndex, assetsList } = this.props;

return this.deleteButtonRefs[assetsList[deletedAssetIndex].id];
```

Reply:
I think I prefer keeping focusAsset. I think it makes it the code clearer, instead of indexing into two separate lists, but I like the prop destructuring.